arm-network/2016-09-01/swagger/virtualNetworkGateway.json

```diff
@@ -1132,7 +1374,39 @@
       },
       "description": "Response for the ListVirtualNetworkGateways API service call."
     },
-	"TunnelConnectionHealth": {
+    "BgpPeerStatusListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BgpPeerStatus"
+          },
+          "description": "List of BGP peers"
+        },
+        "nextLink": {
```

Comment:
NextLink was mistakenly not removed when I figured out pageable didn't work. That was my mistake - should be removed with the next commit.